protoc-gen-gripmock/server.tmpl

```diff
@@ -142,11 +149,12 @@ type payload struct {
 }
 
 type response struct {
-	Data  interface{} `json:"data"`
-	Error string      `json:"error"`
+	Data  interface{} 			`json:"data"`
+	Error string      			`json:"error"`
+	KeepServerStreamOpen bool 	`json:"keepServerStreamOpen"`
```

Comment:
I think we need to put it under `Meta` field, and instead of boolean, it should receive duration.